sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyType.java

```diff
@@ -20,22 +20,22 @@
     public static final KeyType EC = fromString("EC");
```

Comment:
We probably should modify the Javadoc for this as well.

Static value `Encrypt` for KeyType. -> Static value `EC` for KeyType.